src/core/main.cpp

```diff
@@ -567,6 +588,15 @@ int main( int argc, char * * argv )
 		}
 	}
 
+	// Test file argument before continuing
+	if( !fileToLoad.isEmpty() )
+	{
+		fileCheck( fileToLoad );
+	}
+	if( !fileToImport.isEmpty() )
```

Comment:
```c++
else if
```

Should I short this one out if we have one of them and give fileToLoad priority?